packages/gatsby-theme/src/components/Skip/SkipLink.js

```diff
@@ -2,12 +2,13 @@ import React from "react"
 import PropTypes from "prop-types"
 import { Box } from "theme-ui"
 
-const SkipLink = ({ children, id, ...props }) => {
+const SkipLink = ({ children, id, label, ...props }) => {
   return (
     <Box
       as='a'
       tabindex='0'
       href={`#${id}`}
+      ariaLabel={label}
```

Comment:
I think the `ariaLabel` attribute should be kebab case, instead of camel case when used on HTML tags, we could check that please? 🙏

Reply:
Yes, you right! I'm going to change it! 🙏